sleap/nn/inference.py

```diff
@@ -5300,71 +5301,115 @@ def _make_provider_from_cli(args: argparse.Namespace) -> Tuple[Provider, str]:
 
     data_path_obj = Path(data_path)
 
+    # Set output_path_list to None as a default to return later
+    output_path_list = None
+
     # Check that input value is valid
     if not data_path_obj.exists():
         raise ValueError("Path to data_path does not exist")
 
-    # Check for multiple video inputs
-    # Compile file(s) into a list for later itteration
-    if data_path_obj.is_dir():
-        data_path_list = []
-        for file_path in data_path_obj.iterdir():
-            if file_path.is_file():
-                data_path_list.append(Path(file_path))
     elif data_path_obj.is_file():
-        data_path_list = [data_path_obj]
+        # If the file is a CSV file, check for data_paths and output_paths
+        if data_path_obj.suffix.lower() == ".csv":
+            try:
+                data_path_column = None
+                # Read the CSV file
+                df = pd.read_csv(data_path)
+
+                # collect data_paths from column
+                for col_index in range(df.shape[1]):
+                    path_str = df.iloc[0, col_index]
+                    if Path(path_str).exists():
```

Comment:
make sure to explain how data_path is selected in documentation for sleap-track

Reply:
sure thing! Documentation changes are now listed in the original issue. Pasting here for you to verify:

Changes will be made to the sleap-track section of the documentation

positional arguments:
data_path Path to data to predict on. This can be one of the following:

- A .slp file containing labeled data.
- A folder containing multiple video files in supported formats.
- An individual video file in a supported format.
- A CSV file with a column of file paths to video files in supported formats; an optional adjacent column can specify output file paths.
- A text file with a path to a video file on each line.

Reply:
Looks good! Try to keep the same format as the current documentation when you add it. You can add a line to the CSV file documentation like "If more than one columns contains file paths, the first will be used for data path and the next column will be tested as the output path."